demo/main.js

```diff
@@ -949,10 +949,10 @@ shakaDemo.Main = class {
             advanced[drmSystem] = shakaDemo.Main.defaultAdvancedDrmConfig();
           }
           if ('videoRobustness' in params) {
-            advanced[drmSystem].videoRobustness = params['videoRobustness'];
+            advanced[drmSystem].videoRobustness = [params['videoRobustness']];
```

Comment:
Please update the Demo in order to support an array (using ,). See preferredVideoCodecs as example

Reply:
updated